src/plugins/cordova-plugin-inappbrowser/inapp-browser.js

```diff
@@ -0,0 +1,391 @@
+    this._frame.addEventListener('error', this._success.bind(this, InAppBrowser.Events.LOAD_ERROR));
+    this._frame.addEventListener('abort', this._success.bind(this, InAppBrowser.Events.LOAD_ERROR));
+
+    window.addEventListener('resize', this._resizeCallback);
+};
+
+/**
+ * Creates a fake browser's navigation bar, that only contains a button to remove the iframe.
+ * @private
+ */
+IframeBrowser.prototype._createNavigationBar = function () {
+    var navigationDiv = document.createElement('div'),
+        closeSpan = document.createElement('span');
+
+    navigationDiv.style.height = '30px';
+    navigationDiv.style.padding = '2px';
+    navigationDiv.style.backgroundColor = '#404040';
+
+    closeSpan.innerHTML = '✖';
+    closeSpan.style.width = '30px';
+    closeSpan.style.height = '25px';
+    closeSpan.style.color = '#ffffff';
+    closeSpan.style.fontSize = '22px';
+    closeSpan.style.marginLeft = '5px';
+    closeSpan.style.cursor = 'pointer';
+
+    closeSpan.addEventListener('click', function (e) {
+        setTimeout(function () {
+            this.close();
+        }.bind(this), 0);
+    }.bind(this));
+
+    navigationDiv.appendChild(closeSpan);
+
+    return navigationDiv;
+};
+
+/**
+ * Resize the main container with the document.body size, only when it is visible.
+ * @private
+ */
+IframeBrowser.prototype._resizeContainer = function () {
+    if (this._container && this._isVisible) {
+        var body = document.body;
+
+        this._container.style.width = body.clientWidth + 'px';
+        this._container.style.height = body.clientHeight + 'px';
+    }
+};
+
+/**
+ * Execute the given code in the iframe context whenever it's possible. This may
+ * throw an error when the URL loaded in the iframe is not in the same domain than
+ * the application.
+ * @throws It throws an error when the eval call from the iframe fails.
+ * @private
+ */
+IframeBrowser.prototype._injectCode = function (code) {
+    var result = this._frame.contentWindow.eval(code);
+
+    return result ? [result] : [];
+};
+
+/**
+ * Create an instance of InAppBrowser based on the target type. When the type is '_system',
+ * an instance of the system browser is created, when the type is '_blank', an instance of
+ * a browser based on iframe is created. For any other target type, including '_self',
+ * the current window will navigate to the given URL.
+ * @param {function} success
+ * @param {function} fail
+ * @param {Array} args
+ * @return {object}
+ */
+function create(success, fail, args) {
+    // args[0]: url, args[1]: target, args[2]: options
+    var Constructor;
+
+    switch (args[1]) {
+    case '_system':
+        // open in a new browser tab
+        Constructor = SystemBrowser;
+        break;
+    case '_blank':
+        Constructor = IframeBrowser;
+        break;
+    default:
+        // "_self" and any other option, use the same tab
+        window.location = args[0];
```

Comment:
This is wrong. The documentation of inappbrowser plugin is wrong, or I didn't understand it. But after checkign in the mobilespec app what is the expected result for `any other option` and after taking a look at the (android implementation)[https://github.com/apache/cordova-plugin-inappbrowser/blob/master/src/android/InAppBrowser.java#L188], the target `_blank` and `everything else` will be loaded in the InAppBrowser, and for us, the iframe (or a new window after I've added the configuration panel).